src/conventions/h-group/clue-interpretation/connecting-cards.js

```diff
@@ -149,40 +151,122 @@ function find_unknown_connecting(game, giver, target, reacting, identity, connec
 			return;
 		}
 
+		// This may be a bluff even if it matches. resolve_bluff will rule out a bluff if it doesn't
+		// look like it could lead to the following plays.
+		const bluff = game.level >= LEVEL.BLUFFS &&
+			firstPlay &&
+			((giver + 1) % state.numPlayers) == reacting &&
+			!state.hands[reacting].some(c => game.players[reacting].thoughts[c.order].finessed && game.players[reacting].thoughts[c.order].possible.some(id => id.matches(identity)));
```

Comment:
Using `possible.has(identity)` is much faster here. The array support in IdentitySet is mainly for situations where transformations are needed.